src/commands/generate/generators/auth/magic-manifest.js

```diff
@@ -0,0 +1,53 @@
+const manifest = {
+  arcMutations: [
+    { pragma: 'events', item: 'auth-link' },
+  ],
+  sourceFiles: [
+    // API
+    { src: 'handlers/magic/api/auth/example.js', target: 'app/api/auth/example.mjs' },
+    { src: 'handlers/magic/api/auth/login.js', target: 'app/api/auth/login.mjs' },
+    { src: 'handlers/magic/api/auth/signup.js', target: 'app/api/auth/signup.mjs' },
+    { src: 'handlers/magic/api/auth/register.js', target: 'app/api/auth/register.mjs' },
+    { src: 'handlers/magic/api/auth/logout.js', target: 'app/api/auth/logout.mjs' },
+    { src: 'handlers/magic/api/auth/verify.js', target: 'app/api/auth/verify.mjs' },
+    { src: 'handlers/magic/api/auth/welcome.js', target: 'app/api/auth/welcome.mjs' },
+    // Pages
+    { src: 'handlers/magic/pages/auth/example.js', target: 'app/pages/auth/example.mjs' },
+    { src: 'handlers/magic/pages/auth/login.js', target: 'app/pages/auth/login.html' },
+    { src: 'handlers/magic/pages/auth/signup.js', target: 'app/pages/auth/signup.html' },
+    { src: 'handlers/magic/pages/auth/register.js', target: 'app/pages/auth/register.mjs' },
+    { src: 'handlers/magic/pages/auth/verify.js', target: 'app/pages/auth/verify.html' },
+    { src: 'handlers/magic/pages/auth/welcome.js', target: 'app/pages/auth/welcome.mjs' },
+    // Shared code (inside models directory)
+    { src: 'handlers/magic/models/auth/can-i.js', target: 'models/auth/can-i.mjs' },
+    // Events
+    { src: 'handlers/magic/events/auth-link/index.js', target: 'src/events/auth-link/index.mjs' },
```

Comment:
We've changed where events go in Enhance apps. They are now in `jobs/events`. This is going to be a bit of a problem for this part of the generator as it means you also need to add a `src` element to what is written to the arcfile.

Reply:
I don't think it will be too bad. I think you can do:
```
 arcMutations: [
    {
      pragma: 'events', item: `auth-link
  src jobs/events/auth-link` },
  ]
```
The arcMutation method handles multiline items from looking at the tests (i.e. ` let item = '/b\n  method get\n  src foo'`).